ts/Core/Pointer.ts

```diff
@@ -130,6 +131,8 @@ class Pointer {
 
     public hasPinched?: boolean;
 
+    public hasPointerCapture: boolean = false;
+
```

Comment:
Do we need to set it to `false` by default? I think the conditions will work on truthy values.